packages/playwright-test/src/reporters/merge.ts

```diff
@@ -114,25 +115,34 @@ async function mergeEvents(dir: string, shardReportFiles: string[], printStatus:
   const configureEvents: JsonEvent[] = [];
   const beginEvents: JsonEvent[] = [];
   const endEvents: JsonEvent[] = [];
-  const shardEvents = await extractAndParseReports(dir, shardReportFiles, printStatus);
-  shardEvents.sort((a, b) => {
```

Comment:
Let's keep it. We've changed between different naming schemes for files several times and there is a change we'll do it again in the future, so it's better if the merger does not depend on particular naming scheme.